common/lib/xmodule/xmodule/tests/test_xblock_wrappers.py

```diff
@@ -387,6 +388,21 @@ def test_xmodule_handler_return_value(self):
         self.assertIsInstance(response, webob.Response)
         self.assertEqual(response.body, '{}')
 
+    @ddt.data(
+        u'{"test_key": "test_value"}',
+        '{"test_key": "test_value"}',
+    )
```

Comment:
👍 good test data!

Reply:
Thank you @efischer19. I really appreciate your quick review 👍